src/libs/SignatureChecker.sol

```diff
@@ -62,6 +86,23 @@ library SignatureChecker {
         return validateECDSASignature(signer, digest, signature) == Error.NoError;
     }
 
+    /**
+     * @dev    Returns whether an ECDSA/secp256k1 short signature is valid for a signer and digest.
+     * @param  signer The address of the account purported to have signed.
+     * @param  digest The hash of the data that was signed.
+     * @param  r      An ECDSA/secp256k1 signature parameter.
+     * @param  vs     An ECDSA/secp256k1 short signature parameter.
+     * @return isValid Whether the signature is valid.
+     */
```

Comment:
```suggestion
    /**
     * @dev    Returns whether an ECDSA/secp256k1 short signature is valid for a signer and digest.
     * @param  signer  The address of the account purported to have signed.
     * @param  digest  The hash of the data that was signed.
     * @param  r       An ECDSA/secp256k1 signature parameter.
     * @param  vs      An ECDSA/secp256k1 short signature parameter.
     * @return isValid Whether the signature is valid.
     */
```

Reply:
Fixed in this commit: https://github.com/MZero-Labs/common/pull/8/commits/fcb4ccb4fe10e6a5002111e1e3766c056b6bfbec